File: ParkerenIsFijn/ParkingsApp/ParkingsApp.Models/Parkings/Parking.cs
using ParkingsApp.Models.Cities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace ParkingsApp.Models.Parkings
{
    public class Parking : BaseEntity
    {
        [Required]
        public string Title { get; set; }

        public string Address { get; set; }

        [Required]
        public int AvailableCapacity { get; set; }

        [Required]
        public int TotalCapacity { get; set; }

        public City City { get; set; }

        public override string ToString()
        {
            return $"{this.Title}: {this.AvailableCapacity}/{this.TotalCapacity}";
        }
    }
}

File: Parkeren/src/app/models/parking.ts
export default class Parking {
    constructor(
        public id: number,
        public name: string,
        public description: string,
        public isOpen: boolean,
        public availableCapacity: number,
        public totalCapacity: number,
        public address: string
    ){}

    toString(){
        return `${this.name} - ${this.availableCapacity} / ${this.totalCapacity}`;
    }
}
